src/main/java/com/fvv/bookstore/controller/EmployeeControllerImpl.java

```diff
@@ -95,6 +96,29 @@ public Employee findEmployee(final Long id)
 		}
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public List<Employee> listEmployeesByName(final String name) throws PersonNotFoundException, ControllerException {
+		try {
+			return this.employeeDao.listEmployeesByName(name);
+		} catch (DaoException e) {
+			throw new ControllerException("Error to find an employee", e);
+		}
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public Double getSalaryWithCommission(final Double percentage, final Employee employee, final Date date)
```

Comment:
Review this method, I got the wrong calculation, see my comment about that

Reply:
ok